### ISSUE TITLE
Ability to download only certain chapters from a followed series

### ISSUE THREAD

**Issue by jonnyrobbie** [Hamuko/cum#48]
So there is this issue I have. After `follow`ing a series and then listing `chapters`, I see lots of them as 'n'ew (if I understand that correctly). But as far as I know, `download` downloads all chapters from the series. Can I specify only certain chapters to be downloaded? There are multiple releases per chapter as well as a lot of chapters I've already read. I know I could probably use `get` with URL, but with alias, there can be an ambiguity of chapters - for example rozen-maiden-ii has three english releases for c060.

Also it seems that `get` doesn't seem to set the 'downloaded` flag in `chapters`.

**Comment by CounterPillow** [comment]
You can use the `ignore` command to ignore chapters you don't want to download. `get` is for downloading chapters in a way that doesn't touch the database.

**Comment by jonnyrobbie** [comment]
Is there a way to set 'downloaded' flag instead of 'ignore' flag?

**Comment by CounterPillow** [comment]
Unless you download it, no. The downloaded flag is specifically to mean you've downloaded the chapter. We don't track chapter files after downloading them, so you can delete them afterwards if you don't want them.

**Comment by Hamuko** [comment]
Yeah, `cum ignore` is used to not get chapters you don't want. `cum get` by design doesn't do anything to the database, so that you can redownload chapters.

An interactive mode for `cum ignore` could probably help to help ignore the things you don't want. I just need to think of how that would be implemented the best.
